src/main/scala/outwatch/dom/helpers/DomUtils.scala

```diff
@@ -143,27 +142,29 @@ private[outwatch] final case class Receivers(
 
   lazy val observable: Observable[(Seq[Attribute], Seq[IO[StaticVNode]])] = {
     val childStreamReceivers = if (childStreamStatus.hasChildOrChildren) {
-      childNodes.foldRight(Observable.of(List.empty[IO[StaticVNode]])) {
-        case (vn: StaticVNode, obs) => obs.combineLatestWith(BehaviorSubject(IO.pure(vn)))((nodes, n) => n :: nodes)
-        case (csr: ChildStreamReceiver, obs) => obs.combineLatestWith(csr.childStream)((nodes, n) => n :: nodes)
+      childNodes.foldRight(Observable(List.empty[IO[StaticVNode]])) {
+        case (vn: StaticVNode, obs) => obs.combineLatestMap(BehaviorSubject(IO.pure(vn)))((nodes, n) => n :: nodes)
+        case (csr: ChildStreamReceiver, obs) => obs.combineLatestMap(csr.childStream)((nodes, n) => n :: nodes)
         case (csr: ChildrenStreamReceiver, obs) =>
-          obs.combineLatestWith(
-            if (childStreamStatus.hasMultipleChildren) csr.childrenStream.startWith(Seq.empty) else csr.childrenStream
+          obs.combineLatestMap(
+            if (childStreamStatus.hasMultipleChildren) csr.childrenStream.startWith(Seq(Seq.empty)) else csr.childrenStream
           )((nodes, n) => n.toList ++ nodes)
       }
     } else {
-      Observable.of(Seq.empty)
+      Observable(Seq.empty)
     }
 
     // only use last encountered observable per attribute
-    val attributeReceivers: Observable[Seq[Attribute]] = Observable.combineLatest(
+    val attributeReceivers: Observable[Seq[Attribute]] = Observable.combineLatestList(
       attributeStreamReceivers
         .groupBy(_.attribute)
         .values
-        .map(_.last.attributeStream)(breakOut)
+        .map(_.last.attributeStream)(breakOut): _*
     )
 
-    attributeReceivers.combineLatest(childStreamReceivers)
+    attributeReceivers.startWith(Seq(Seq.empty)).combineLatest(
+      childStreamReceivers.startWith(Seq(Seq.empty))
+    ).dropWhile { case (a, c) => a.isEmpty && c.isEmpty }
```

Comment:
What is this `dropWhile` good for?

Reply:
Without doing `startwith(Seq(Seq.empty))` and `dropWhile`, the `combinedLatest` would emit an element only when *both* the `attributeReceivers` and `childStreamReceivers` emit. This makes it emit when *either* of them emits.

Reply:
So the behavior of combineLatest is different in monix?

Reply:
That's what I thought too, but from reading the documentation now, it seems that `combineLatest` 
 in rxjs should behave the same as in Monix. Let me look into why I had to do this (or why it's not needed with rxjs/rxscala).

Reply:
The difference is between the  `Observable.combineLatest` rxscala companion object method and the Monix `Observable.combineLatestList`. The Monix `combineLatestList` doesn't emit an empty list if the argument list is empty, rxscala does.

Reply:
Should we have a test for this?

Reply:
Now the code doesn't do `.startWith(Seq(Seq.empty))....dropWhile { case (a, c) => a.isEmpty && c.isEmpty }` any more, instead it emits an `Observable(Seq.empty[Attribute])` if the `attributeReceivers` list is empty (just like in the rxscala version). There are several tests that fail if we don't do that.